be/src/storage/tablet.h

```diff
@@ -450,6 +454,8 @@ class Tablet : public BaseTablet {
     // another tablet with the same tablet id
     // currently, it will be used in Restore process
     bool _will_be_force_replaced = false;
+
+    std::atomic<bool> _is_dropping{false};
```

Comment:
Is it necessary to add a new state?

Reply:
> Is it necessary to add a new state?

When drop and clone operations are concurrent, different execution sequences can lead to various outcomes, such as the clone succeed while the apply stop. In fact, during the drop tablet process, once the drop operation begins, any subsequent clone requests on this tablet are meaningless. Therefore, directly rejecting these requests would be better and can also avoid complex concurrency issues.